packages/console/src/console-history/ConsoleHistoryItem.tsx

```diff
@@ -18,8 +18,10 @@ const log = Log.module('ConsoleHistoryItem');
 interface ConsoleHistoryItemProps {
   item: ConsoleHistoryActionItem;
   language: string;
-  openObject: (object: VariableDefinition) => void;
+  openObject(object: VariableDefinition): void;
   disabled?: boolean;
+  supportsType(type: string): boolean;
+  iconForType(type: string): ReactElement;
```

Comment:
I prefer the other format for TS function definitions, and what was there previously:
```
openObject: (object: VariableDefinition) => void;
```
Are we using the other format anywhere else? Figure we should be consistent, unsure about a linter rule for this.

Reply:
I think I've picked the shorthand in other spots. Turns out there is a slight difference in that TS is more strict on the property version (what you prefer) due to how it handles variance.

Here's the linter rule. Want me to make a ticket to enable it?

https://typescript-eslint.io/rules/method-signature-style/

Reply:
#1578